src/Splitter.sol

```diff
@@ -38,6 +38,7 @@ interface FarmLike {
 contract Splitter {
     mapping (address => uint256) public wards;
     FlapLike    public           flapper;
+    uint256     public           live; // Active Flag
```

Comment:
Not really with a strong reason but I'd put the `live` flag right after `wards`.

Reply:
Done